src/crypto/core.rs

```diff
@@ -0,0 +1,47 @@
+#[cfg(all(feature = "aws_lc_rs", not(any(target_arch = "wasm32", target_os = "windows"))))]
+pub(crate) mod dep {
+    pub(crate) use ::aws_lc_rs::{constant_time, error, hmac, rand};
+
+    pub(crate) mod signature {
+        pub(crate) use ::aws_lc_rs::signature::*;
+
+        #[inline]
+        pub(crate) fn ecdsa_key_pair_from_pkcs8(
+            alg: &'static EcdsaSigningAlgorithm,
+            pkcs8: &[u8],
+            _rng: &dyn ::aws_lc_rs::rand::SecureRandom,
+        ) -> Result<EcdsaKeyPair, ::aws_lc_rs::error::KeyRejected> {
+            EcdsaKeyPair::from_pkcs8(alg, pkcs8)
```

Comment:
Hello! -- I stumbled across this thread, and thought I would provide a little context around this difference.

In ring v0.16.x, `EcdsaKeyPair::from_pkcs8` only [takes two parameters](https://docs.rs/ring/0.16.20/ring/signature/struct.EcdsaKeyPair.html#method.from_pkcs8). But in ring 0.17, this function [takes three parameters](https://docs.rs/ring/latest/ring/signature/struct.EcdsaKeyPair.html#method.from_pkcs8). Because aws-lc-rs v1.x maintains compatibility with ring 0.16.x, our function (likewise) only takes two parameters.

Unfortunately, I don't see a way we could support a 3rd parameter w/o breaking backwards compatibility. But feel free to drop us [an issue](https://github.com/aws/aws-lc-rs/issues/new/choose) if you have questions or would like to suggest an improvement to our API. Thanks!